x/rate/semaphore_test.go

```diff
@@ -8,11 +8,11 @@ import (
 	"testing"
 	"time"
 
-	"github.com/go-redis/redis/v8"
 	"github.com/google/uuid"
-	"github.com/qqunity/asynq"
-	"github.com/qqunity/asynq/internal/base"
-	asynqcontext "github.com/qqunity/asynq/internal/context"
+	"github.com/platacard/asynq"
+	"github.com/platacard/asynq/internal/base"
+	asynqcontext "github.com/platacard/asynq/internal/context"
+	"github.com/redis/go-redis/v9"
```

Comment:
Had to update redis to v9 because this `x` submodule is outdated in upstream, so a few minor changes are introduced here.